### PR TITLE
Update NewsFeedManagerPage.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191 
added else condition for archived feeds

File: tests/system/webdriver/tests/components/NewsFeedManager0002Test.php [hunk 1]
```diff
@@ -147,23 +147,23 @@ public function setFilter_TestFilters_ShouldFilterTags2()
 
 		$this->newsFeedManagerPage->addFeed($feedName_1);
 		$message = $this->newsFeedManagerPage->getAlertMessage();
-		$this->assertTrue(strpos($message, 'Banner successfully saved') >= 0, 'Banner save should return success');
+		$this->assertTrue(strpos($message, 'Banner successfully saved') >= 0, 'news feed save should return success');
```
Comment (puneet0191) [inline tests/system/webdriver/tests/components/NewsFeedManager0002Test.php:150]: When you are adding a News Feed, why are you asserting with Banner successfully saved?

Comment (puneet0191) [inline tests/system/webdriver/tests/components/NewsFeedManager0002Test.php:150]: why are you using strpos?, when we can exactly match the messages and use assertEquals

**Comment by puneet0191** [comment]
You need to redo this one

**Pull request closed without merging**